src/Symfony/Component/Security/Guard/GuardAuthenticatorHandler.php

```diff
@@ -52,7 +52,7 @@ public function __construct(TokenStorageInterface $tokenStorage, EventDispatcher
     /**
      * Authenticates the given token in the system.
      */
-    public function authenticateWithToken(TokenInterface $token, Request $request/*, string $providerKey */)
+    public function authenticateWithToken(TokenInterface $token, Request $request)
     {
         $providerKey = \func_num_args() > 2 ? func_get_arg(2) : null;
```

Comment:
The argument is used, we should rather uncomment it on 5.x (with null default value to preserve the current behavior).

Reply:
So we have to close this request?

Reply:
Or you can rebase your branch on the 5.x branch (and change the base branch of your PR accordingly), then uncomment the argument as I explained before.

Reply:
Comment were removed after 4.2 Branch.

Reply:
This comment is the usual way how we document a new parameter that is to be introduced with the next major release.

Reply:
Then what happens if this parameter is removed again in the next version? Will there be a comment again?